### PR TITLE
Add travel/venue information

### PR DESCRIPTION
**Pull request by mrmundt**
## Description

This PR adds the initial information about the venue and nearby hotels, plus other travel information (e.g, driving/parking instructions).

Preview of Travel Information Page

## Motivation and Context
We want to inform people of details about the venue.

## Checklist:

- [x] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2023-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @mrmundt @cabejackson @manning-ncsa @jmelot @jbteves @jsubida

### CODE REVIEW

**Comment by manning-ncsa** [comment]
Looks good from a quick build in terms of formatting (I have not reviewed the content). In the process I noticed that if you click Attend in the nav bar and/or Attend > Register you see "We’ll post information when it becomes available.". Perhaps that message should be removed from the Attend page and replaced with a link to the Register and Travel pages.

File: pages/attend/travel.md [hunk 1]
```diff
@@ -4,7 +4,78 @@ title: Travel
...
+menubar_toc: true
 set_last_modified: true
 ---
 
-We'll post information when it becomes available.
+
+## Venue Information
+
+- **Dates**: October 16-18, 2023
+- **Location**: University of Illinois in Chicago – [Student Center East](https://meetings.uic.edu/facilities/student-center-east/) – 750 S. Halsted, Chicago, IL 60607
```
Comment (sci-tab) [inline pages/attend/travel.md:15]: Is this a good place to add this information? https://usrse.slack.com/archives/C04E9M6NW7P/p1679922252568479 

Mostly interested in ADA and this link https://oae.uic.edu/resources/gender-inclusive-resources/

Comment (mrmundt) [reply pages/attend/travel.md:15]: I'd say that's entirely up to you. Where would you like that information?

**Comment by mrmundt** [comment]
> Looks good from a quick build in terms of formatting (I have not reviewed the content). In the process I noticed that if you click Attend in the nav bar and/or Attend > Register you see "We’ll post information when it becomes available.". Perhaps that message should be removed from the Attend page and replaced with a link to the Register and Travel pages.

Good call! Fixed!

File: pages/attend/attend.md [hunk 2]
```diff
@@ -7,4 +7,11 @@ permalink: attend/
 set_last_modified: true
 ---
 
-We'll post information when it becomes available.
+## Register
+
+See the [Registration page]({{ site.baseurl }}/attend/travel) for more details.
```
Comment (manning-ncsa) [inline pages/attend/attend.md:12]: Oops, looks like copypasta. You probably want /attend/register here.

File: pages/attend/travel.md (hunk 1 above)
Comment (sci-tab) [reply pages/attend/travel.md:15]: I think it might be! Let me confirm with the other co-chairs

File: pages/attend/attend.md (hunk 2 above)
Comment (mrmundt) [reply pages/attend/attend.md]: Siiigggghhh. Copy-pasta failed me again. Fixed.

Comment (manning-ncsa) [reply pages/attend/attend.md]: Fixed

**Comment by manning-ncsa** [comment]
Looks fine to me. I'm curious what the `google.ca` addresses for the maps are about. How is `google.ca` different from `google.com`?

File: pages/attend/travel.md (hunk 1 above)
Comment (sci-tab) [reply pages/attend/travel.md:15]: Yes, that's perfect @mrmundt !! I just confirmed this is a good place to include that. Thanks for taking my question and implementing it

**Comment by mrmundt** [comment]
> Looks fine to me. I'm curious what the `google.ca` addresses for the maps are about. How is `google.ca` different from `google.com`?

Not a clue!

**Review by sci-tab — APPROVED** [review]

**Comment by mrmundt** [comment]
> Looks fine to me. I'm curious what the `google.ca` addresses for the maps are about. How is `google.ca` different from `google.com`?

@manning-ncsa  - Jk, I looked it up - that's the Canadian-specific domain that shows more Canadian-specific results. I changed it to the generic `google.com` URL. (Why it was the Canadian one - no idea. It was a copy-pasta from the original document as well.)

**Review by jbteves — APPROVED** [review]
One minor comment but looks good to me.

File: pages/attend/travel.md [hunk 3]
```diff
@@ -4,7 +4,87 @@ title: Travel
...
+- ADA Accessbility
+  - Elevators available to all floors of Student Center East
+  - If special accommodations are required, please contact the organizers at [usrsecon2023@easychair.org](mailto:usrsecon2023@easychair.org)
+- [Gender Inclusive Resources](https://oae.uic.edu/resources/gender-inclusive-resources/)
+  - Gender neutral facilities: 2nd and 8th Floor of Student Center East
+  - Lactation room: Rooms 243B and 243C of Commuter Student Resource Center
+
+## Hotel Accommodation
+
+We’re reserving hotel room blocks at several nearby hotels. Please check back
```
[the remark this answers is not in the document]
Comment (mrmundt) [reply pages/attend/travel.md:28]: Unfortunately, I have no idea.

**Pull request merged**